clients/osv/src/main/kotlin/OsvApiClient.kt

```diff
@@ -114,12 +115,14 @@ data class VulnerabilitiesForPackageBatchResponse(
     @Serializable
     data class IdList(
         @SerialName("vulns")
-        val vulnerabilities: List<Id> = emptyList()
+        val vulnerabilities: List<Vulnerability> = emptyList()
     )
 
     @Serializable
-    data class Id(
-        val id: String
+    data class Vulnerability(
```

Comment:
How about naming this differently, because we have a class of same name in `Model.kt` ?
Maybe `VulnerabilityRef` ? Or `IdWithTimestamp`?

Reply:
You already approved this as part of https://github.com/oss-review-toolkit/ort/pull/5565 😆